erc-hl-nicks.el

```diff
@@ -354,7 +354,7 @@
 ;; For first time use
 ;;;###autoload
 (when (and (boundp 'erc-modules)
-           (not (member 'hl-nicks 'erc-modules)))
+           (not (member 'hl-nicks erc-modules)))
```

Comment:
Why bother with this check? `add-to-list` already checks for presence in the list before adding it